src/backend/test/2_api.spec.js

```diff
@@ -1,538 +1,610 @@
-process.env.NODE_ENV = 'test';
+process.env.NODE_ENV = "test";
 
-const chai = require('chai');
+const chai = require("chai");
 const should = chai.should();
-const chaiHttp = require('chai-http');
-const server = require('../app');
-const knex = require('../db/knex.js');
-const queries = require('../db/queries.js');
-const io = require('socket.io-client')
-const socketUrl = 'http://localhost:3000/game';
-const options ={
+const chaiHttp = require("chai-http");
+const server = require("../app");
+const knex = require("../db/knex.js");
+const queries = require("../db/queries.js");
+const io = require("socket.io-client");
+const socketUrl = "http://localhost:3000/game";
+const options = {
   // resource: 'game',
   newConnection: true,
-  transports: ['websocket']
+  transports: ["websocket"]
 };
 
 chai.use(chaiHttp);
 
-const asyncHOF = (fn) => {
+const asyncHOF = fn => {
   return done => {
     fn.call().then(done, err => {
       done(err);
     });
   };
 };
 
-describe('API Routes', _ => {
-
-  describe('/api/games', _ => {
-    before( async () => {
-      try{
+describe("API Routes", _ => {
```

Comment:
these tests dont really need to run actual database queries since you already do so above.



you can stub the actual routes using sinon or jest spys and just test that the routes forward parameters correctly